docs/software/python.md

```diff
@@ -0,0 +1,389 @@
+---
+tags:
+ - Software
+ - python
+---
+# Installing Python Packages 
+
+There are a few different ways to install Python packages. Each ORCD system has its own set of Python modules and naming conventions for those, along with a set of recommendations for installing Python packages. This page is meant to give a general overview and link to those pages.
+
+=== "Engaging"
+
+    [Engaging Installing Python Packages Documentation](https://engaging-web.mit.edu/eofe-wiki/software/python_packages/)
+    
+=== "Satori"
+
+    [Satori Installing Python Packages Documentation](https://mit-satori.github.io/satori-using-anaconda.html)
+
+=== "SuperCloud"
+
+    [SuperCloud Installing Python Packages Documentation](https://mit-supercloud.github.io/supercloud-docs/software-packages/#python-packages)
+
+=== "OpenMind"
+
+    [OpenMind Installing Python Packages Documentation](https://github.mit.edu/MGHPCC/OpenMind/wiki/How-to-make-Python-ready-for-use%3F)
+
+Python packages will need to be installed in your home directory or other directory you have write access to. There are a few different ways to do this, each with its own pros and cons. At a high level, you can:
```

Comment:
I think it would be beneficial if you could include in bold or other emphasis to read through the different ways of implementation and pros/cons to determine which structure is the best. Many tend to use the first method that comes up because they assume it's the simplest, and/or people not familiar with Python might resort to using the "simplest" way of using the home directory, which can cause version issues down the road. It doesn't hurt to add a disclaimer (maybe a drop-down on why choosing the right method is important?)